packages/dd-trace/test/datastreams/processor.spec.js

```diff
@@ -0,0 +1,177 @@
+'use strict'
+
+require('../setup/tap')
+
+const { hostname } = require('os')
+const Uint64 = require('int64-buffer').Uint64BE
+
+const { LogCollapsingLowestDenseDDSketch } = require('@datadog/sketches-js')
+
+const HIGH_ACCURACY_DISTRIBUTION = 0.0075
+
+const pkg = require('../../../../package.json')
+const DEFAULT_TIMESTAMP = Number(new Date('2023-04-20T16:20:00.000Z'))
+const DEFAULT_LATENCY = 100000000
+const DEFAULT_PARENT_HASH = Buffer.from('e858292fd15a41e4', 'hex')
+const ANOTHER_PARENT_HASH = Buffer.from('e858292fd15a4100', 'hex')
+const DEFAULT_CURRENT_HASH = Buffer.from('e858212fd11a41e5', 'hex')
+const ANOTHER_CURRENT_HASH = Buffer.from('e851212fd11a21e9', 'hex')
+
+const writer = {
+  flush: sinon.stub()
+}
+const DataStreamsWriter = sinon.stub().returns(writer)
```

Comment:
This stays permanently stubbed for the duration of the test suite. Can you move all of this into a `before` / `beforeEach` block and then add the `.restore` in the equivalent `after` / `afterEach` block?

Reply:
Also, you may need to wrap all of these describes into a single overarching describe for the entire file, with the before/after inside of the top level describe.

Reply:
Sorry, I'm wrong, there's nothing to restore here